assets/TechItEasy.sql

```diff
@@ -0,0 +1,82 @@
+DROP TABLE IF EXISTS television;
+DROP TABLE IF EXISTS remote_controller;
+DROP TABLE IF EXISTS ci_module;
+DROP TABLE IF EXISTS wall_bracket;
+DROP TABLE IF EXISTS users;
+
```

Comment:
je mist nog een 'tussen tabel' television_wall_brackets waar je alle combi van tv's en wall_brackets (id's) kan stoppen (wanneer het om may-to-many association gaat).